azure-ts-dynamicresource/index.ts

```diff
@@ -0,0 +1,95 @@
+// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+import * as pulumi from "@pulumi/pulumi";
+import * as azure from "@pulumi/azure";
+
+import { CDNCustomDomainResource } from "./cdnCustomDomain";
+
+/**
+ * The location where our resource group and the resources under it will be created.
+ *
+ * To externalize this value, and make this configurable across environments/stacks,
+ * learn more at https://pulumi.io/reference/config/.
+ */
+const location = "West US";
+
+/** The custom domain host of the CDN endpoint. */
+export let cdnCustomDomainResource: CDNCustomDomainResource | undefined;
```

Comment:
why declare this up here.  you can just move down to the place you actually create it.

Reply:
Yep. Moved it.